lib/services/common-api-presenter.js

```diff
@@ -209,28 +211,24 @@ function CommonApiPresenterFactory(
             var scope = self.res.locals.scope;
 
             var promises = [
-                lookupService.ipAddressToMacAddress(self.res.locals.ipAddress),
-                templates.get(name, scope)
+                Promise.props({
+                    server: configuration.get('apiServerAddress', '10.1.1.1'),
+                    port: configuration.get('apiServerPort', 80),
+                    ipaddress: self.res.locals.ipAddress,
+                    netmask: configuration.get('dhcpSubnetMask', '255.255.255.0'),
+                    gateway: configuration.get('dhcpGateway', '10.1.1.1'),
+                    macaddress: lookupService.ipAddressToMacAddress(self.res.locals.ipAddress),
+                    sku: Env.get('config', {}, [ scope[0] ]),
```

Comment:
is scope guaranteed to be an array/list? Looks like it is in `lib/services/http-service.js` but wanted to check to see if it was fiddled with elsewhere...